app/assets/javascripts/modules/auto_logoff.js.coffee

```diff
@@ -15,7 +15,7 @@ class app.AutoLogoff
 
   logoff_timer = () ->
 
-    if document.URL.indexOf('session/new') > -1
+    if document.URL.indexOf('session/new') > -1 || document.URL.indexOf('session/sso/inactive') > -1
```

Comment:
create own method for this ?

isLoginPage ?

Reply:
:+1: